src/app/containers/MostRead/utilities/processMostRead.js

```diff
@@ -89,6 +92,9 @@ const mostReadItems = ({ data, numberOfItems }) => {
     }
     return items;
   }
+  logger.warn(MOST_READ_STALE_DATA, {
+    message: 'Most read lastUpdatedTimestamp value is greater than 35min',
```

Comment:
Is it worth including the value of the lastUpdatedTimestamp? Or the delta (i.e. how much greater than 35 minutes is it?)

Reply:
Added the lastUpdatedTimestamp value to the message.